terraform/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default     = "prod"
```

Comment:
Very minor, but could be worth considering specifying the `type` for each of these variables explicitly - when not specified any type can be passed through, but given that all of these are strings I suspect there's not much space for confusion as it is